create_bfb

```diff
@@ -159,7 +159,7 @@ GRUB_DEFAULT=saved
 GRUB_DISABLE_SUBMENU=true
 GRUB_TERMINAL=console
 GRUB_SERIAL_COMMAND="serial --unit=1"
-GRUB_CMDLINE_LINUX="crashkernel=auto  console=hvc0 console=ttyAMA0 earlycon=pl011,0x01000000 modprobe.blacklist=mlx5_core,mlx5_ib"
+GRUB_CMDLINE_LINUX="crashkernel=auto  console=hvc0 console=ttyAMA0 earlycon=pl011,0x01000000 modprobe.blacklist=mlx5_core,mlx5_ib net.ifnames=0 biosdevname=0"
```

Comment:
Why was this needed out of curiosity?

Reply:
This change came from Vladimir

Reply:
This is a link to the Mellanox/mariner repo https://github.com/Mellanox/bfb-mariner

Reply:
This is the feedback I got: "we use udev rules to set network interfaces names. Parameters above prevent from other utilities control over network interfaces names"